packages/web/src/features/languages/ManageLanguageView.tsx

```diff
@@ -1,30 +1,33 @@
 import apiClient from '../../shared/apiClient';
 import View from '../../shared/components/View';
 import ViewTitle from '../../shared/components/ViewTitle';
-import { FormEvent } from 'react';
 import { LoaderFunctionArgs, useLoaderData } from 'react-router-dom';
+import { useForm } from 'react-hook-form';
 import TextInput from '../../shared/components/TextInput';
 import FormLabel from '../../shared/components/FormLabel';
-import Button from '../../shared/components/Button';
 import { GetLanguageResponseBody } from '@translation/api-types';
 import { useTranslation } from 'react-i18next';
+import Form from '../../shared/components/Form';
+import InputError from '../../shared/components/InputError';
+import SubmitButton from '../../shared/components/SubmitButton';
 
 export async function manageLanguageViewLoader({ params }: LoaderFunctionArgs) {
   return apiClient.languages.findByCode(params.code ?? 'unknown');
 }
 
+interface FormData {
+  name: string;
+}
+
 export default function ManageLanguageView() {
   const language = useLoaderData() as GetLanguageResponseBody;
 
   const { t } = useTranslation();
 
-  async function onSubmit(e: FormEvent<HTMLFormElement>) {
-    e.preventDefault();
-    const name = (
-      e.currentTarget.elements.namedItem('name') as HTMLInputElement
-    ).value;
+  const formContext = useForm<FormData>();
```

Comment:
The view that has the form will call `useForm` to make all of the helpers available in the view